src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -2197,7 +2197,7 @@ std::string FusedOpsCodeGenerator::GetJitLoad(const FusedOpsConfiguration& conf,
 
             if (vec_size > 1) {
                 return block_read;
-            } else if (input_tensor.LogicalSize() > 1) {
+            } else if (input_tensor.LogicalSize() > 1 || input_tensor.is_dynamic()) {
```

Comment:
considering the comment below("in such scenario we can safely load sub_group_size elements from the pointer"), I'm afraid this may have issue when input_tensor size becomes 1 from dynamic shape. Did you review that?

Reply:
@isanghao Yes, I already tested 1 element of dynamic shape input case. I added those test case in single layer test.

Reply:
aligned offline to check further

Reply:
@isanghao Updated the condition for when any one of static dimension element has more than one element.